SIG.md

```diff
@@ -0,0 +1,107 @@
+# Cilium SIGS
+
+The Cilium project has Special Interest Groups, or SIGs, with a common purpose of advancing the project with respect to a specific topic, such as network policy or documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. 
+
+Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:
+
+- Policy
+- Scalability
+- Docs
+
+Each SIG must have a charter that specifies its scope (topics, subsystems, code repos, and/or directories), responsibilities, areas of authority, and basic governance. See the SIG charter process for details on how charters are managed. SIGs are relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-project processes (e.g., the release process) and assets (e.g., the cilium repo).
+
+A primary reason that SIGs exist is as forums for collaboration. Much work in a SIG should stay local within that SIG. However, SIGs must communicate in the open, ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions, and periodically communicate a high-level summary of the SIG's work to the community.
+
+# SIG Roles
+## SIG Lead
+
+- Membership tracked in sigs.yaml
+- MUST be an odd number of SIG Leads in a group.
+- MUST be a reviewer of one or more areas of the project, and should be a subject matter expert in their SIG area.
+- SHOULD define how priorities and commitments are managed. MAY delegate to other leads as needed.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MUST identify, track, and maintain the metadata of the SIGs enhancements for the current release and serve as point of contact for the release team, but MAY delegate to other contributors to fulfill these responsibilities.
+- MUST organize a main group meeting and make sure sigs.yaml is up to date.
```

Comment:
I thought we were leaving SIG meetings as optional? i.e. any given SIG can decide whether to have meetings, or just a Slack channel